CriticalMass/SettingsSection.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SettingsSection.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 19.04.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+
+extension Hashable where Self: CaseIterable {
+    var index: Self.AllCases.Index {
+        return type(of: self).allCases.firstIndex(of: self)!
+    }
+}
+
+enum Section: Int, CaseIterable {
+    case gps
+    case darkMode
```

Comment:
In my opinion, darkmode and gps should be a model of the preference section instead of representing there own section.

Reply:
I updated it to be in preferences